packages/studio-ui/src/hooks/useInjectActiveStyles.tsx

```diff
@@ -25,7 +25,7 @@ export default function useInjectUserStyles(
     if (!iframeDocument) {
       return;
     }
-    userStyleImports.forEach((styleFilepath) => {
+    activeStylesFilepaths.forEach((styleFilepath) => {
       injectStyleIntoIframe(iframeDocument, styleFilepath);
```

Comment:
This isn't really a question about this PR, but just to refresh my understanding here, now that we're manually going through dependency trees to get all the CSS filepaths and dynamically importing them, why do we still need the Vite plugin to add the `inline` query param? Is that something we could just do when making the dynamic import?

Reply:
When we dynamically import components, they may also depend CSS. So, Vite will try to add style tags when traversing file dependencies if it comes across a style file, so we want to make sure that we add the inline tag to any style file that Vite touches.  This is also generally the case for transitive dependencies.  If we imported something with the `inline` query param, it won't be propagated when vite imports that file's dependencies so we'd need to add it ourselves.